synapse/models/inet.py

```diff
@@ -23,7 +23,7 @@
 logger = logging.getLogger(__name__)
 drivre = regex.compile(r'^\w[:|]')
 fqdnre = regex.compile(r'^[\w._-]+$', regex.U)
-srv6re = regex.compile(r'^\[([a-f0-9\.:]+)\]:(\d+)$')
+srv6re = regex.compile(r'^\[([a-f0-9\.:]+)\](?::(\d+))?$', regex.IGNORECASE)
```

Comment:
This is used in the Addr norm on line 133, which expects if it matches that there will be a port value, but that is not always the case now, should probably tweak that a bit.